src/applications/vaos/services/appointment/transformers.js

```diff
@@ -179,13 +193,32 @@ function getAppointmentDuration(appt) {
  * @param {Object} appt  VAR appointment object
  * @returns {Array} Array of participants of FHIR appointment
  */
-function addParticipantAndContained(appt, transformed) {
-  const isVideo = isVideoVisit(appt);
-  let participant;
-  let contained;
+function setParticipant(appt) {
+  if (isCommunityCare(appt)) {
+    const address = appt.address;
+    return [
+      {
+        actor: {
```

Comment:
Based on the spec, `actor` is always a reference, so we wouldn't ever see this info in here.

We don't really know how these locations will come through, but I think the easiest thing for us to do right now is put this in the `contained` array, as a Location resource. That's a plausible way of implementing this, and doesn't force us to deal with bundled Locations.

Reply:
Gotcha.  Moved this to the `contained` array for now.